torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -3149,3 +3149,20 @@ void TensorExprKernel::runKernel(Stack& stack) {
     push_one(stack, std::move(o));
   }
 }
+
+void TensorExprKernel::runFast(
+    const std::vector<void*>& inputs,
+    const std::vector<void*>& outputs) {
+  KernelScope kernelScope(&kernelArena_);
+
+  std::vector<void*> args(inputs);
```

Comment:
Micro-optimization: do `args.reserve(inputs.size() + outputs.size() + constants_.size())` so we don't end up resizing this thing multiple times.

Reply:
Good idea! If we want to optimize it even further, we could've returned the preallocated vector and than at the call-site we would need to only overwrite the elements corresponding to inputs/outputs pointers. But I decided not to go that far with the first version